### ISSUE TITLE
Generated grpc bindings not reflecting on current releases

### ISSUE THREAD

**Issue by franklinselva** [aiplan4eu/unified-planning#224]
**Describe the bug**

The grpc bindings generated based on `unified_planning.proto` are not reflecting on pypi releases. 

**To Reproduce**

1. `pip install unified-planning grpcio grpcio-tools`
2. `python -c "from unified_planning.grpc.generated import unified_planning_pb2_grpc"`

**Actual behavior**

Throws,

```bash
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/up-env/lib/python3.8/site-packages/unified_planning/grpc/generated/unified_planning_pb2_grpc.py", line 5, in <module>
    import unified_planning_pb2 as unified__planning__pb2
ModuleNotFoundError: No module named 'unified_planning_pb2'
```

The above issue seems to be not the case if `unified_planning` is installed through git.

**Comment by alvalentini** [comment]
This issue was fixed by @arbimo in #125.
After that fix we haven't released a new stable version yet, but if you install a pre-release version (using the `--pre` option) it should work.

**Comment by franklinselva** [comment]
@alvalentini Thank you.
